versions.sh

```diff
@@ -69,9 +69,7 @@ for version in "${versions[@]}"; do
 			variants: [
 				(
 					"buster",
-					if env.rcVersion == "2.6" then
-						"stretch"
-					else empty end
+                                        "bullseye"
```

Comment:
These should be in the opposite order, and ought to have matching whitespace :sweat_smile:

```suggestion
					"bullseye",
					"buster"
```

Reply:
oh, whitespaces :facepalm: 
So AFAIU we want bullseye to be the default one?

Reply:
Yep!